canonical-websites/www.ubuntu.com
# Remove /download/cloud/autopilot

## Done
Remove `/download/cloud/autopilot` page. Created a redirect back to `/download/cloud`. Removed it from the navigation.

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/download/cloud/](http://0.0.0.0:8001/download/cloud/)
- Check that the navigation doesn't have Autopilot
- Go to [http://0.0.0.0:8001/download/cloud/autopilot](http://0.0.0.0:8001/download/cloud/autopilot) and check it redirect

## Issue / Card
Fixes https://github.com/canonical-websites/www.ubuntu.com/issues/2189
